### PR TITLE
Add fixtures for ZE50

### PR DESCRIPTION
**Pull request by PysX**
Some fixtures for ZE50  (contribution for #48)

Some additional information : 

- `charge-history` returns `err.func.403` (Access is denied for this resource)
- `hvac-history` returns `err.tech.501`  (This feature is not technically supported by this gateway)
- `hvac-sessions` returns `err.tech.501` (This feature is not technically supported by this gateway)
- `hvac-status` returns `err.func.403` (Operation not supported for this can (C1A))

Although hvac returns errors, i am able to start hvac from Renault App and from Home assistant with hassRenaultZE services.

Others actions may be exist. On Renault App, i can search a location and send it to the car.

### CODE REVIEW

File: tests/fixtures/kamereon/error/hvac-status-error.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    }
+  ],
+  "errors": [
+    {
+      "errorCode": "err.func.403",
+      "errorMessage": "{\"errors\":[{\"status\":\"403\",\"code\":\"403011\",\"title\":\"Operation not supported\",\"detail\":\"Operation not supported for this can (C1A)\"}]}"
+    }
+  ],
+  "error_reference": "FUNCTIONAL"
+}
```
Comment (epenet) [inline tests/fixtures/kamereon/error/hvac-status-error.json]: EOF, and please rename "hvac-status-error.json" > "hvac-status.403.403011.json"

File: tests/fixtures/kamereon/vehicle_data/cockpit_zoe2.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "data": {
+    "type": "Car",
+    "id": "VF1AAAAA555777999",
+    "attributes": {
+      "fuelAutonomy": 0,
+      "fuelQuantity": 0,
+      "totalMileage": 5785.75
+    }
+  }
+}
```
Comment (epenet) [inline tests/fixtures/kamereon/vehicle_data/cockpit_zoe2.json]: EOF, and please rename "cockpit_zoe2.json" > "cockpit.zoe50.json"

File: tests/fixtures/kamereon/vehicles/zoe2_withnav.json [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        "vin": "VF1AAAAA555777999",
+        "lastModifiedDate": "2020-11-29T22:01:21.162572Z",
+        "brand": "RENAULT",
+        "startDate": "2020-08-21",
+        "ownershipStartDate": "2020-01-13",
+        "ownershipEndDate": "2020-08-21"
+      }
+    ],
+    "accountId": "account-id-1"
+  }
```
Comment (epenet) [inline tests/fixtures/kamereon/vehicles/zoe2_withnav.json]: EOF, and please rename "zoe2_withnav.json" > "zoe_50.1.json"

Comment (PysX) [reply tests/fixtures/kamereon/vehicles/zoe2_withnav.json]: All files renamed and fixed !

**Comment by epenet** [comment]
Thanks a lot @PysX

**Pull request merged**